pkg/client/registry.go

```diff
@@ -0,0 +1,17 @@
+package client
```

Comment:
We can rename the package as registry. As registry is specific, and we can have other clients as well, like a kubernetes client. Can use different packages for each client.